package.json

```diff
@@ -82,60 +91,29 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "~7.3.4",
+    "@babel/core": "~7.12.1",
+    "@babel/preset-env": "~7.12.1",
+    "@babel/preset-react": "~7.12.1",
+    "@babel/cli": "~7.12.1",
+    "@babel/register": "~7.12.1",
     "@material-ui/core": "^4.8.0",
     "@material-ui/icons": "^4.0.0",
-    "@svgr/webpack": "^4.3.3",
-    "@webpack-cli/init": "^0.2.2",
-    "babel-cli": "~6.26.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-named-asset-import": "^0.3.4",
-    "babel-preset-env": "~1.7.0",
-    "babel-preset-react-app": "~7.0.2",
-    "babel-register": "~6.26.0",
-    "case-sensitive-paths-webpack-plugin": "2.2.0",
-    "chalk": "~2.4.2",
+    "babel-loader": "~8.1.0",
     "css-loader": "~2.1.1",
     "eslint-plugin-react-hooks": "^4.1.2",
-    "extract-text-webpack-plugin": "~3.0.2",
-    "file-loader": "3.0.1",
-    "fork-ts-checker-webpack-plugin-alt": "0.4.14",
-    "fs-extra": "7.0.1",
-    "html-webpack-plugin": "4.0.0-alpha.2",
-    "identity-obj-proxy": "3.0.0",
-    "jest": "~24.5.0",
-    "jest-pnp-resolver": "1.2.1",
-    "jest-resolve": "24.5.0",
-    "js-cookie": "~2.2.0",
+    "jest": "~26.6.2",
+    "jest-pnp-resolver": "~1.2.2",
+    "jest-resolve": "~26.6.2",
+    "js-cookie": "~2.2.1",
     "mini-css-extract-plugin": "0.5.0",
     "node-sass": "~4.14.1",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
-    "pnp-webpack-plugin": "1.4.1",
-    "popper.js": "~1.14.7",
-    "postcss-flexbugs-fixes": "4.1.0",
-    "postcss-loader": "3.0.0",
-    "postcss-preset-env": "6.6.0",
-    "postcss-safe-parser": "4.0.1",
```

Comment:
For packages like these, how did you determine that they weren't being used? I'm not asking because I'm suspicious, I'm just trying to learn : )

Reply:
I just looked up the purpose of a package in `npmjs.com` and tried to look up the reference in the code and googled about the packages, ran the build and tested. I just went by rationale is build failed for missing package, runtime error, ran through couple of browsers and If none happened then I removed it. 

As you mentioned we might need to see as we all start using new version of `package.json` and find error due to this.